contestants.json

```diff
@@ -80,5 +80,12 @@
             "github" : "https://github.com/aayushjoglekar",
             "twitter" : "https://github.com/aayushjoglekar"
         }
+        {
+        "name" : "Anubhav Marwaha",
+        "imageurl" : "",
```

Comment:
this parameter is required, please upload an image.

Reply:
okay

Reply:
done!